packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Overview of the mobile Identity Engine SDK
+---
+
+<div class="oie-embedded-sdk">
+
+Design the implementation of the sign-in flow for your mobile app by understanding the representation of the flow, and with examples of the common parts.
+
+<ApiLifecycle access="ie" /><br>
+
+## Introduction
+
+Integrate the Okta sign-in flow into your app in one of two ways: redirect the request to Okta which handles the flow in its own window, or handle each step in the flow by building the appropriate user interface elements, capturing the information, and updating Okta.
+
+This guide is an overview of building your own user interface. For information on the redirect model, see <StackSnippet snippet="redirectquickstart" inline />.
+
+## Sign-in flow
+
+Okta supports many ways of authenticating the identity of a user during the sign-in flow. An Okta org administrator creates _policies_, different mixes of authentication methods (also called factors), and assigns them to apps, people, groups, and more. Policies also configure how many authentication methods are required (multifactor), and which methods are optional. This results in a large number of possible combinations for the different factors and steps to complete the sign-in flow.
+
+The Android and iOS Identity Engine SDKs represent the sign-in flow as a state machine. You initialize the machine with the details of your Okta Application Integration, request the initial step in the flow, and cycle through responding to steps until either the user is signed in, cancels, or an error occurs.
+
+<div class="common-image-format">
+
+![A diagram showing the sign-in flow.](/img/mobile-sdk/mobile-idx-basic-flow.png "A diagram that shows the sign-in flow.")
+
+</div>
+
+Each sign-in step may include one or more possible user actions, such as:
+
+- Choosing an authenticator, such as Okta Verify or security questions.
+- Entering a One-Time Passcode (OTP).
+- Cancelling the sign-in flow.
+
+## Sign-in objects
+
+Each step in the sign-in flow is represented by a number of different SDK objects:
+
+<div class="common-image-format">
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+- **Response:** The top-level object that represents a step and contains all the other objects. It also indicates a successful sign-in, and is used to send messages for cancelling the sign-in flow, or to retrieve an access token after the sign-in flow succeeds. A response may contain multiple authenticators and remediations.
```

Comment:
The sentences are talking about additional functional of the Response object. From that perspective "can" doesn't really make sense because the Response object does those things... "can" implies it might rather than it does. Any other suggestions?

Reply:
Again it's a pasive voice thing (is used to) so I was working around that. How about

It will also indicate a successful sign-in, send messages to cancel the sign-in flow, and retrieve an access token after the sign-in flow succeeds.

I know it's a future tense thing, but its the best I have off the top fo my head.

Reply:
I used this for the next revision:

> It includes a property that indicates a successful sign-in and functions for cancelling the sign-in flow, or retrieving the access token after the sign-in flow succeeds.